text/0113-secure-script-container.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Secure Script Container
+status: DRAFTED
+created_at: 2020-07-15
+pr: https://github.com/salesforce/lwc-rfcs/pull/38
+champion: Caridy Patiño, Ted Conn
+---
+
+# Secure Script Container for LWC Apps
```

Comment:
How are we ensuring that the scripts are indeed secure? If we aren't doing anything to secure the script (eg: an allow list of approved scripts) then we should possibly make this `<lwc-script>` or something similar?